SFS_time_domain/driving_function_imp_nfchoa.m

```diff
@@ -89,7 +89,7 @@
 
 % Compute impulse responses of modal filters
 dm = [repmat(pulse,[order+1 1]) zeros(order+1,N-length(pulse))];
-for n=2:order+1
+for n=1:order+1
```

Comment:
I guess this change has some undesired consequences. Please try (I used octave):
```Matlab
>> conf = SFS_config;
>> x0 = secondary_source_positions(conf);
>> driving_function_imp_nfchoa(x0,[0 2.5 0],'ps',conf);
error: driving_function_imp_nfchoa_ps: A(I,J): row index out of bounds; value 1 out of bound 0
error: called from
    driving_function_imp_nfchoa_ps at line 100 column 20
    driving_function_imp_nfchoa at line 100 column 13
```

If you switch back to master and run the same commands you won't get an error.